src/AzureDeprecation.APIs.REST/Contracts/DashboardInfo.cs

```diff
@@ -0,0 +1,7 @@
+namespace AzureDeprecation.APIs.REST.Contracts;
+
+public class DashboardInfo
```

Comment:
I don't think we still need this, do we?

Reply:
Yeah, thanks